frontend/app/services/TouchpointBackend.scala

```diff
@@ -33,19 +37,20 @@ object TouchpointBackend {
     TouchpointBackend(touchpointBackendConfig)
   }
 
-  def apply(touchpointBackendConfig: TouchpointBackendConfig): TouchpointBackend = {
-    val stripeService = new StripeService(touchpointBackendConfig.stripe, new TouchpointBackendMetrics with StatusMetrics {
-      val backendEnv = touchpointBackendConfig.stripe.envName
+  def apply(backend: TouchpointBackendConfig): TouchpointBackend = {
+    val stripeService = new StripeService(backend.stripe, new TouchpointBackendMetrics with StatusMetrics {
+      val backendEnv = backend.stripe.envName
       val service = "Stripe"
     })
 
-    val zuoraSoapService = new ZuoraSoapService(touchpointBackendConfig.zuora)
-    val zuoraRestService = new ZuoraRestService(touchpointBackendConfig
-      .zuora.copy(url = touchpointBackendConfig.zuoraRestUrl(Config.config)))
+    val restBackendConfig = backend.zuora.copy(url = backend.zuoraRestUrl(Config.config))
 
-    val memberRepository = new FrontendMemberRepository(touchpointBackendConfig.salesforce)
+    val zuoraSoapClient = new ClientWithFeatureSupplier(FeatureChoice.codes, backend.zuora, backend.zuoraMetrics("zuora-soap-client"), Akka.system())
```

Comment:
not having a package name here is not consistent with the naming convention recently adopted